codegen/src/Utils.scala

```diff
@@ -33,13 +39,13 @@ object Utils {
       if (pulumiPackage.language.java.packages.view.nonEmpty) {
         pulumiPackage.language.java.packages.view.mapValues { pkg =>
           pkg.split("\\.").filter(_.nonEmpty).toSeq
-        }.toMap
+        }.toMap.withDefault(slashModuleToPackageParts) // fallback to slash mapping
```

Comment:
I don't know why, but before this change, we got key not found error instead of the fallback function

Reply:
why wouldn't we?

Reply:
IDK, I just fond this one confusing